src/test/java/org/cactoos/collection/NoNullsTest.java

```diff
@@ -38,42 +38,45 @@
  */
 public final class NoNullsTest {
 
-    /**
-     * A rule for handling an exception.
-     */
-    @Rule
-    public final ExpectedException exception = ExpectedException.none();
-
     @Test
     public void throwsErrorIfNullInToArray() {
-        this.exception.expect(IllegalStateException.class);
-        this.exception.expectMessage(
-            "Item #1 of #toArray() is NULL"
+        MatcherAssert.assertThat(
+            "Must throw exception",
+            () -> new NoNulls<>(
+                new CollectionOf<>(1, null, 3)
+            ).toArray(),
+            new Throws<>(
+                "Item #1 of #toArray() is NULL",
+                IllegalStateException.class
+            )
         );
-        new NoNulls<>(
-            new CollectionOf<>(1, null, 3)
-        ).toArray();
     }
 
     @Test
     public void throwsErrorIfNullInToArrayWithArg() {
-        this.exception.expect(IllegalStateException.class);
-        this.exception.expectMessage(
-            "Item #1 of #toArray(array) is NULL"
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`